libs/muddle/internal/peer_selector.hpp

```diff
@@ -73,7 +73,7 @@ class PeerSelector : public core::PeriodicRunnable
   {
     PeerData    peer_data{};
     std::size_t peer_index{0};  // The current peer being evaluated
-    Timepoint   last_updated{Clock::now()};
+    std::size_t consecutive_failures{0};
```

Comment:
don't we avoid size_t? not that it matters in this case

Reply:
In general no, It gets painful when interacting with standard containers. The only place where we really don't want to use it is in the VM

Reply:
and serialization stuff right

Reply:
Yep that is correct